Server/EVESharp.Node/Services/Dogma/dogmaIM.cs

```diff
@@ -222,21 +210,35 @@ public PyDataType LogAttribute (ServiceCall call, PyInteger itemID, PyInteger at
     public PyDataType Activate (ServiceCall call, PyInteger itemID, PyString effectName, PyDataType target, PyDataType repeat)
     {
         ShipModule module = this.Items.GetItem <ShipModule> (itemID);
-
         EffectsManager.GetForItem (module, call.Session).ApplyEffect (effectName, call.Session);
-
         return null;
     }
 
     public PyDataType Deactivate (ServiceCall call, PyInteger itemID, PyString effectName)
     {
         ShipModule module = this.Items.GetItem <ShipModule> (itemID);
-
         EffectsManager.GetForItem (module, call.Session).StopApplyingEffect (effectName, call.Session);
+        return null;
+    }
 
+    // === NEW STUBS ADDED BELOW ===
+
+    public PyDataType CheckSendLocationInfo(ServiceCall call)
+    {
+        Console.WriteLine("[dogmaIM] CheckSendLocationInfo called");
```

Comment:
This should be logged automatically by the service handler, so this logging should go. Also please use the logging classes instead of writting to the console like this